bin/GameIndex.yaml

```diff
@@ -18350,6 +18350,9 @@ SLPM-60106:
 SLPM-60107:
   name: "Golf Paradise"
   region: "NTSC-J"
+SLPM-60109:
+  name: "RIDGE RACER V [Trial]"
```

Comment:
Ridge Racer V, no need to capitalise it. Example: http://www.thecoverproject.net/view.php?cover_id=4420

Reply:
Done :+1: